### ISSUE TITLE
Automated a11y failure on new footer

### ISSUE THREAD

**Issue by maureenlholland** [mozilla/bedrock#15662]
## Description

<img width="1546" alt="Screenshot 2024-12-04 at 9 43 54 AM" src="https://github.com/user-attachments/assets/11b46fe2-50c0-48dc-98e0-f5ebec0c3713">

See https://bedrock.readthedocs.io/en/latest/testing.html#running-playwright-tests for how to run tests and get feedback on failure

---

#### Success Criteria
- [ ] No automated accessibility failure on new footer

**Comment by maureenlholland** [comment]
will add `aria-label` to distinguish footer navigations as primary & secondary

<img width="572" alt="Screenshot 2024-12-04 at 9 49 09 AM" src="https://github.com/user-attachments/assets/baa695ca-afa0-49a5-9d50-a83355ffd1c1">
